core/trino-main/src/main/java/io/trino/MoreSizeOfMain.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino;
+
+import com.google.common.collect.Multimap;
+import io.airlift.stats.Distribution;
+import io.airlift.units.DataSize;
+import io.airlift.units.Duration;
+import org.joda.time.DateTime;
+
+import java.util.AbstractMap;
+import java.util.Map;
+import java.util.function.ToLongFunction;
+
+import static io.airlift.slice.SizeOf.instanceSize;
+import static io.airlift.slice.SizeOf.sizeOfObjectArray;
+
+public final class MoreSizeOfMain
```

Comment:
unusual name. MoreSizeOf would be enough.

Reply:
there is another MoreSizeOf. if the method names were not same, we could rely on static imports.